langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -220,6 +312,10 @@ public isolated function toBytes(string str) returns byte[] = @java:Method {
 
 # Constructs a string from its UTF-8 representation in bytes.
 #
+# ```ballerina
+# string:fromBytes([72, 101, 108, 108, 111, 32, 66, 97, 108, 108, 101, 114, 105, 110, 97, 33]) ⇒ Hello, World!
```

Comment:
Shall we add an example for an error scenario?

Reply:
yes, makes sense. Will add.